frappe/core/doctype/user/user.json

```diff
@@ -581,14 +582,21 @@
    "fieldtype": "Password",
    "label": "API Secret",
    "read_only": 1
+  },
+  {
+   "fieldname": "reset_key_created_on",
```

Comment:
use `creation` date... no need for extra field

Reply:
creation date?

Reply:
My bad...

`expire_reset_password_key` that you are using below... but looks like it is missing in User doctype... You need to add it but rename it to `reset_password_key_expiry`

Reply:
fixed.